### ISSUE TITLE
execv error: no such file or directory

### ISSUE THREAD

**Issue by peterweissbier** [mmtrt/foobar2000_AppImage#3]
i am only getting the error with this appimage. other appimages work.

$ ./foobar2000-2.0_WP-x86_64.AppImage 
execv error: No such file or directory
$ sudo chmod +x foobar2000-2.0_WP-x86_64.AppImage 
[sudo] password for kaatayra: 
$ ./foobar2000-2.0_WP-x86_64.AppImage 
execv error: No such file or directory

**Comment by mmtrt** [comment]
can you retest again it should be working now.

**Comment by peterweissbier** [comment]
apologies, i tried installing this on amd64 archlinux. i just saw its for aarch64 which i did not know is actual arm. 

i tried really hard with any method like flatpak, wine and snap to make foobar2000 working with a THEME on linux. it seems impossible. any version i try will not load this theme https://github.com/Ottodix/Eole-foobar-theme

do you have any suggestions on how i could make it work?

**Comment by torkus** [comment]
It looks like that theme doesn't work on 64bit builds of foobar2000:

![Screenshot at 2024-06-15 08-03-32](https://github.com/mmtrt/foobar2000_AppImage/assets/48790775/e4bf7e63-0c68-46c7-b1d3-21247f42bb18)

And it looks like this AppImage is using 64bit foobar2000:
https://github.com/mmtrt/foobar2000_AppImage/blob/b0e62db70f2ee5923402beca5eba87504bb65cf9/deploy.sh#L22

**update**: actually, it looks like it gets rid of the 64bit downloads with this trailing command at the end:

    rm *x64*.exe *64ec*.exe 

So just ignore me ;)

**Comment by peterweissbier** [comment]
yeah i know that its just for the 32 bit and some of the versions of foobar2000. thanks for the info. which files is this appimage based on? can you try making that theme work in your version?
